tests/gamma/csm_observability_test.py

```diff
@@ -50,7 +50,8 @@ def test_csm_observability(self):
         #   resource creation out of self.startTestServers()
         with self.subTest("1_run_test_server"):
             test_server: _XdsTestServer = self.startTestServers(
-                enable_csm_observability=True
+                enable_csm_observability=True,
```

Comment:
Should we add this to an override of [`RegularXdsKubernetesTestCase.startTestServers`](https://github.com/grpc/psm-interop/blob/292cea23d6263862416008b0a76b4c0c16cbaf69/framework/xds_k8s_testcase.py#L834) within [`GammaXdsKubernetesTestCase`](https://github.com/grpc/psm-interop/blob/292cea23d6263862416008b0a76b4c0c16cbaf69/framework/xds_gamma_testcase.py#L35), similar to how `startTestClient` currently has a GAMMA-specific override. This way, all future CSM tests will get this and we won't have to go through this debugging process again in the future.

Reply:
Good point - but do we actually need to set `generate_mesh_id` in all gamma tests? If so, then we should make `GammaXdsKubernetesTestCase.startTestServers()` to set it automatically. If we don't (f.e. in affinity / baseline) tests, I think we should keep it as is.

Reply:
Yes. We do need `generate_mesh_id` in all GAMMA tests.

Reply:
Ok, will try to refactor. Thanks for the confirmation.